Dockerfile.dev

```
@@ -24,7 +24,7 @@ ADD ${REMOTE_SOURCE}/Makefile ${REMOTE_SOURCE}/main.go ${REMOTE_SOURCE}/go.mod $
 RUN make build
 
 #@follow_tag(registry.redhat.io/ubi8:latest)
-FROM docker.io/centos:8 AS centos
+FROM registry.redhat.io/ubi8/ubi:8.7-929
```

Sorry for taking so long to look at this. By now the version has changed:

```suggestion
FROM registry.redhat.io/ubi8:8.7-1090
```